bin/generate_driver

```diff
@@ -118,6 +118,16 @@ EOS
 end
 
 # -----------------------------------------
+# machine resource basics.
```

Comment:
That's a somewhat WIP thing where I was about to start experimenting with running unit specs on the generated driver. I can remove it for this PR, though probably no one should be looking at this file anyway.